mrp_bom_hierarchy/__openerp__.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Eficent (<http://www.eficent.com/>)
+#              <contact@eficent.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "MRP BOM Hierarchy",
+    "version": "1.0",
+    "author": "Eficent",
+    "website": "www.eficent.com",
+    "category": "Manufacturing",
+    "depends": ["mrp"],
+    "description": """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+MRP BOM Hierarchy
+=================
+
+This module was written to extend the functionality of Bill of
+Materials to support users to better maintain the BOM hierarchy.
+
+This module adds a new menu entry 'Bill of Materials Hierarchy' in
+'Manufacturing | Products', from which the user can create a complete BOM
+hierarchy from the tree view.
+
+The user can now search using the field 'Full BOM Hierarchy Reference' to find
+all the BOM hierarchy associated to a particular BOM Reference at once.
+
+The user can now search using the field 'Full BOM Hierarchy Name' to find
+all the BOM hierarchy associated to a particular BOM Name at once.
+
```

Comment:
Maybe easier to read with:
The user can now search using the field 'Full BOM Hierarchy Reference' (or Name) to find
all the BOM hierarchy associated to a particular BOM Reference (or Name) at once.